### PR TITLE
Add status to Env

### PR DESCRIPTION
**Pull request by psav**

### CODE REVIEW

File: apis/cloud.redhat.com/v1alpha1/clowdenvironment_types.go [hunk 1]
```diff
@@ -418,6 +418,7 @@ type MinioStatus struct {
 type ClowdEnvironmentStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
+	Conditions      []ClowdCondition        `json:"conditions,omitempty"`
```
Comment (BlakeHolifield) [inline apis/cloud.redhat.com/v1alpha1/clowdenvironment_types.go:421]: I see that the root(ish) type has changed. We can change the `status` structs as we please, correct? We aren't tied to the same rigidity as the spec?

**Review by BlakeHolifield — APPROVED** [review]
I don't see any issues and this is so so valuable. Do we need unit testing around these? I don't think there's a good way to test this in `kuttl`.

File: apis/cloud.redhat.com/v1alpha1/clowdenvironment_types.go (hunk 1 above)
Comment (psav) [reply apis/cloud.redhat.com/v1alpha1/clowdenvironment_types.go:421]: This is an addition - and it's omit-empty

**Comment by kylape** [comment]
Merged manually

**Pull request closed without merging**